static/app/views/replays/detail/trace/trace.tsx

```diff
@@ -147,4 +172,18 @@ const OverflowScrollBorderedSection = styled(BorderedSection)`
   }
 `;
 
+function getTraceStatus(traceState: ExternalState) {
```

Comment:
Only used in a single place it seems, I would inline it.

Reply:
Agree.